html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form>
+    <label>Email:</label><br>
+      <input type="text" name="username" size="15" maxlength="30" />
+<br>
+    <label> Why is coding cool?</label><br>
+    <select name="devices">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+
+<br>
+    <label>
+      Favorite Language (choose one):
+    </label>
+      <br>
+      <input type="radio" name="genre" value="rock" checked="checked" /> HTML
+      <input type="radio" name="genre" value="pop" />
+      CSS
+      <input type="radio" name="genre" value="jazz" />
+      JavaScript
+<br>
+      <label>
+      Would you like to sign up for more information?
+    </label>
+      <br>
+      <input type="checkbox" name="service" value="itunes" checked="checked" /> Sign me Up!
+
+    <label>
```

Comment:
Your label elements need `for` attributes to reference the form control element.